addon/templates/components/nav-route.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if isVisible}}
   {{#if route}}
-    {{#frost-link route disabled=true}}
+    {{#frost-link route disabled=true hook=(concat hook '-link')}}
```

Comment:
What happens if there are multiple 'route' menu items defined in a nav category? How would one distinguish them via the hook?